src/FirstSteps.Tests/RPG/Adventures/MineTests.cs

```diff
@@ -23,19 +23,23 @@ public void Dwarf_Should_Gain_One_Coin_After_the_First_Visit__In_The_Mine()
         [InlineData(4, 2)]
         [InlineData(5, 3)]
         [InlineData(8, 4)]
-        [InlineData(9, 5)]
-        [InlineData(12, 6)]
-        public void Dwarf_Should_Gain_Three_Coins_When_Numbers_Are_Odd_Otherwise_Should_Gain_One_Coin(int expectedCoins, int visitCounter)
+        [InlineData(11, 5)]
+        [InlineData(14, 6)]
+        [InlineData(15, 7)]
+        [InlineData(18, 8)]
+        [InlineData(19, 9)]
+        [InlineData(22, 10)]
```

Comment:
The tenth visit is even and divisible by five. So based on the description you should add 5 coins. 3 because it is even, and an extra 2 because it is divisible by five. 

- Every even visit: **3 coins**
- Every visit divisible by five: **extra 2 coins** to the normal amount